src/core/mapping/instance_manager.h

```diff
@@ -54,10 +54,10 @@ std::ostream& operator<<(std::ostream& os, const RegionGroup& region_group);
 
 struct InstanceSet {
  public:
+  using RegionGroupP = std::shared_ptr<RegionGroup>;
```

Comment:
what's the reason for this bogus change?

Reply:
No real reason, leftover from a change I reverted. I will undo this.